docs/sops/serverless-controller-knative-up.md

```diff
@@ -0,0 +1,93 @@
+# OpenShift Serverless Knative Serving Control Plane Readiness
```

Comment:
This seems like it is targeting only the operator pods. So that is not really "Serving", right? 
For Serving we'd also need to look at all pods in `knative-serving`, `knative-serving-ingress` and (when istio is used) `istio-system` (or a custom namespace where istio is deployed).

Reply:
Down in the file, there are instructions to check if pods in `knative-serving` is running.

But, let me add the same thing for other namespaces.

Thanks

Reply:
Added some text for Ingress now.

Thanks @ReToCode for the suggestion.